ContentfulPersistenceSwift.podspec

```diff
@@ -28,6 +28,6 @@ Pod::Spec.new do |spec|
   spec.watchos.deployment_target = '2.0'
   spec.tvos.deployment_target    = '9.0'
 
-  spec.dependency 'Contentful', '~> 0.7.5'
+  spec.dependency 'Contentful', '~> 0.7.7'
```

Comment:
My pattern matcher throw an error here: Should'nt it be `0.8.0`? Since this is what you wrote in the _changelog_? 🤔